hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -810,8 +820,8 @@ protected static S3AStorageStatistics createStorageStatistics(
   }
 
   /**
-   * Verify that the bucket exists. This does not check permissions,
-   * not even read access.
+   * Verify that the bucket exists.
```

Comment:
was the original probe, retained because 3rd party stores don't always handle v2.
now we have changed the default probe to "0", we could think about cutting v1 entirely